lib/Pipeline.js

```diff
@@ -82,12 +83,23 @@ Pipeline.processJSON = function(gltf, options) {
  */
 Pipeline.processJSONWithExtras  = function(gltfWithExtras, options) {
     options = defaultValue(options, {});
+
+    addDefaults(gltfWithExtras, options);
+
+    // Print statistics for unoptimized input
+    if (options.stats) {
+        if (options.stats) {
+            console.log("Statistics for " + options.inputPath + "\n------------------\n" +
+                        getStatistics(gltfWithExtras).toString());
+        }
+    }
```

Comment:
Oops. Muscle memory.

Yes, I thought of that, but since the old `printStatistics` was a single call, I kept it single as a style thing.

Will change and force push right away.